workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/Codegen/VisualElementConcealer.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEngine.UIElements;
+
+namespace Improbable.Gdk.Debug.WorkerInspector.Codegen
+{
+    public class VisualElementConcealer : IConcealable
```

Comment:
If this is the only implementation of `IConcealable` we can probably drop the interface and use the concrete type everywhere unless we have a use case in mind?